### PR TITLE
CBL2 and FMP new models, objects and improvements to profiles

### PR DESCRIPTION
**Pull request by jgomezr**
CBL2 and FMP new models and improvements to profiles, classes and test

### CODE REVIEW

File: src/classes/LACCoffeeCKWFarmers.cls [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    this.stdController = controller;
+    this.coffeeCKW = (LAC_Coffee_CKW__c)controller.getRecord();   
+                                                 
+       LAC_Coffee_CKW__c[] ckwIds = [Select LAC_Farmer_Details__r.Contact__c From LAC_Coffee_CKW__c Where Id =:coffeeCKW.Id];
+        if (ckwIds.size() > 0) {
+            ckwId = ckwIds[0];
+        }
+        }
+
+    public LAC_Coffee_CKW__c getCkwId() {
```
Comment (lc0rp) [inline src/classes/LACCoffeeCKWFarmers.cls]: Please out-dent this to be in line with the others

File: src/classes/LACCoffeeCKWFarmers.cls [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+public LACCoffeeCKWFarmers(ApexPages.StandardController controller) {
+    this.stdController = controller;
+    this.coffeeCKW = (LAC_Coffee_CKW__c)controller.getRecord();   
+                                                 
+       LAC_Coffee_CKW__c[] ckwIds = [Select LAC_Farmer_Details__r.Contact__c From LAC_Coffee_CKW__c Where Id =:coffeeCKW.Id];
+        if (ckwIds.size() > 0) {
+            ckwId = ckwIds[0];
+        }
+        }
```
Comment (lc0rp) [inline src/classes/LACCoffeeCKWFarmers.cls]: This bracket is out of line. It should be in-line with "public"

File: src/classes/MyApexClass.cls [hunk 3]
```diff
@@ -0,0 +1,15 @@
+public with sharing class MyApexClass {
+
+	private final sObject mysObject;
```
Comment (lc0rp) [inline src/classes/MyApexClass.cls]: Out of line.

File: src/classes/cooperative_AreaFarmersChart.cls [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                                Where cooperativeProfile__c =:cooperative.Id][0].get('fArea');
+        if (maxFarmArea == null) maxFarmArea = 0;
+    }
+
+    /**
+    *Get an area list related to a farmers that belongs a specific cooperative
+    *
+    */
+    public List<LAC_Farmer_Details__c> getAreaFarmers() {
+        List<LAC_Farmer_Details__c> AreaFarmers= [Select Farmer_Code__c,
```
Comment (lc0rp) [inline src/classes/cooperative_AreaFarmersChart.cls]: variables should start with small laters: areaFarmers not AreaFarmers. Classes would start with capital letters.

**Comment by lc0rp** [comment]
Pretty good. What's the code coverage?  - Luke.

**Comment by lc0rp** [comment]
It also seems there are some conflicts. You need to merge in the development branch, solve conflicts locally, and then re-create the pull request.

**Comment by jgomezr** [comment]
hi luke the code coverage for  cooperative_AreaFarmersChartTest.cls is 90% and LACCoffeeCKWFarmersTest  is 95%, today fix de code indentation.

**Comment by jgomezr** [comment]
Changes make to LACCoffeeCKWFarmers to improve line identation and change on cooperative_AreaFarmersChart on line 31 change name of variable

**Comment by jgomezr** [comment]
Hi luke, the modifications to this pull request are done, plis check and merge

**Pull request merged**